### DISCUSSION TITLE
Maintain scene orientation

### DISCUSSION THREAD

**Discussion by briancantwe** [pierotofy/OpenSplat#54] [General]
Hello again!  I'm now having a great time processing my own data with OpenSplat!  Very smooth sailing so far.

Question.  I have a known scene orientation (in Colmap) that I would like to preserve if possible, but the training results are coming out in at a different location/scale.  Is there an option to maintain the input orientation?  

Thanks again!

**Comment by pierotofy** [comment]
#34 we'd welcome contributions!

**Comment by pierotofy** [comment]
#56 should take care of this.

**Comment by briancantwe** [comment]
Amazing!  Thank you!
